### PR TITLE
docs: fix typos in CHANGELOG/CHANGELOG-1.25.md

### PR DESCRIPTION
**Pull request by songxiao-wang87**
What type of PR is this?
/kind cleanup

What this PR does / why we need it:
none

Which issue(s) this PR fixes:
none

Special notes for your reviewer:
none

Does this PR introduce a user-facing change?
none

```release-note
NONE
```

### CODE REVIEW

**Comment by songxiao-wang87** [comment]
@yangjunmyfm192085

**Review by yangjunmyfm192085 — COMMENTED** [review]
/ok-to-test

File: CHANGELOG/CHANGELOG-1.25.md [hunk 1]
```diff
@@ -1003,7 +1003,7 @@ ConfigMap rendering [issue](https://github.com/kubernetes/kubernetes/issues/1120
...
 - Etcd: Update to v3.5.4 ([#110033](https://github.com/kubernetes/kubernetes/pull/110033), [@mk46](https://github.com/mk46)) [SIG API Machinery, Cloud Provider, Cluster Lifecycle and Testing]
 - Faster mount detection for linux kernel 5.10+ using openat2 speeding up pod churn rates. On Kernel versions less 5.10, it will fallback to using the original way of detecting mount points i.e by parsing /proc/mounts. ([#109217](https://github.com/kubernetes/kubernetes/pull/109217), [@manugupt1](https://github.com/manugupt1))
-- FibreChannel volume plugin may match the wrong device and wrong associated devicemapper parent. This may cause a disater that pods attach wrong disks. ([#110719](https://github.com/kubernetes/kubernetes/pull/110719), [@xakdwch](https://github.com/xakdwch))
+- FibreChannel volume plugin may match the wrong device and wrong associated devicemapper parent. This may cause a disaster that pods attach wrong disks. ([#110719](https://github.com/kubernetes/kubernetes/pull/110719), [@xakdwch](https://github.com/xakdwch))
```
Comment (yangjunmyfm192085) [inline CHANGELOG/CHANGELOG-1.25.md:1006]: There is another one in changelog 1.25 that also needs to be modified

**Comment by xmudrii** [comment]
/release-note-edit
```release-note
NONE
```

**Review by xmudrii — APPROVED** [review]
/lgtm
/approve

**Pull request merged**